docs/sources/mimir/release-notes/v2.13.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Grafana Mimir version 2.13 release notes"
+menuTitle: "V2.13 release notes"
+description: "Release notes for Grafana Mimir version 2.13"
+weight: 1100
+---
+
+# Grafana Mimir version 2.13 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.13 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Store-gateway can be configured to explicitly disable or enable tenants**. This is useful in cases when compaction slows down for a tenant to temporarily exclude a tenant while compaction catches up without impact other tenants.
+
+- Configurable S3 bucket lookup type improves interoperability with **S3-compatible providers such as **Tencent and Alibaba\*\*\*\*. Configure lookup type via `-<prefix>.s3.bucket-lookup-type` or `-common.storage.s3-bucket-lookup-type`.
+
+- **Mimirtool can now verify the validity of a Mimir runtime configuration** file with the `mimirtool runtime-config verify` command.
+
+- **Configuration of TLS for S3 buckets** is now possible via `-<prefix>.s3.http.*` and `-common.storage.s3.http.*` configuration options.
+
+- **Improved OTLP ingestion performance** by using native translation, which reduces memory usage in the distributor by up to 30% and CPU usage by up to 8%.
+
+- **Active series are now updated along with owned series.** This means the number of active series for a tenant is much more accurate after adding ingesters.
```

Comment:
Might be worth explaining more why this is useful to a user / operator - doesn't this mean that there should be fewer issues with limits during ingester scaling, for example?

Reply:
yeah, fair, how is https://github.com/grafana/mimir/pull/8445/commits/c29fbf4fdf861a1b440f4d8edd54a59861afd260?

Reply:
LGTM, thanks.